.env.example

```diff
@@ -0,0 +1,3 @@
+PORT=8000
+DB_URL=LINK TO MONGODB
```

Comment:
If in the example you add any values, it should be example values, defaults or no values. In this case example or no values wouls be good